styles.css

```diff
@@ -0,0 +1,87 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  background: #f7f7f9;
+}
+
+.all {
+  height: 916px;
+  top: 1px;
+  border-radius: 1px;
+}
+
+header {
+  height: 72px;
+  width: 100vw;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  width: 80%;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 50px 0 0 0;
+  padding: 114px 24px 0 24px;
+  min-height: 626px;
+  height: 100%;
+  min-width: 375px;
+  width: 100%;
+  justify-content: center;
+  background-image: url("./images/Header bg@2x.png");
+  background-repeat: no-repeat;
+  border-radius: 0 0 0 90px;
+  background-size: 100% 100%;
+  margin-bottom: 100px;
+}
+
+.content-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
- [ ]  Great job with your project 👏. As the previous reviewers mentioned, kindly ensure your design is responsive and can be seen. Kindly fix this 🙏 . One of the reasons your design isn't clear and responsive is because of the width given to almost all elements.

Also, you can give a height of 100vh to your main section on this part of the project, so your section will fit until the 100vh(view height) of the screen that you're using 🥇